packages/botonic-core/index.d.ts

```diff
@@ -1 +1,2 @@
+// eslint-disable-next-line node/no-unpublished-import
```

Comment:
this file doesn't make sense at all ATM. We defined explicitly to export all definitions from 'src', but this will be automatically added when compiling from TS to JS.